README.md

```diff
@@ -43,7 +45,10 @@ $ curl http://localhost:8080/status
 }
 ```
 
-**/cas/empty_blob_sha6** (returns OK, useful to test a bazel-remote instance is running and accepting requests)
+**/cas/e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855**
+
+Returns OK. The empty CAS blob is always available, even if the cache is empty. This can be used to test that
```

Comment:
Could you remove the "Returns OK." sentence, and squash this down do a single commit? Otherwise, lgtm.

Reply:
done!